app/src/main/java/com/example/minwoo/pingpongchat_client/LoginActivity.java

```diff
@@ -134,19 +134,20 @@ private void handleSignInResultOnStart(GoogleSignInResult result) {
                 // Signed in Google successfully, try to sign in to Server.
                 String personName = account.getDisplayName();
                 String personEmail = account.getEmail();
-                String personId = account.getId();
                 String personIdToken = account.getIdToken();
+                String personPhotoUrl = account.getPhotoUrl().toString();
                 JsonObject personData = new JsonObject();
+                personData.addProperty("name", personName);
                 personData.addProperty("email", personEmail);
                 personData.addProperty("token", personIdToken);
+                personData.addProperty("photo", personPhotoUrl);
 
                 Log.i("GoogleLogin", "personName=" + personName);
                 Log.i("GoogleLogin", "personEmail=" + personEmail);
-                Log.i("GoogleLogin", "personId=" + personId);
                 Log.i("GoogleLogin", "personIdToken=" + personIdToken);
+                Log.i("GoogleLogin", "personPhotoUrl=" + personPhotoUrl);
```

Comment:
please refactor handleSignInResultOnStart() and handleSignInResult() since they have same logic. you can remove duplcate code by making new method for this.